File: arrayManipulation.py
def arrayManipulation(n, queries):
    arr = [0]*n
    for a,b,k in queries:
        arr[a-1] += k
        if b < n: arr[b] -= k
    
    c = mx = 0
    for e in arr:
        c += e
        if c > mx: mx = c

    return mx

File: arrayManipulation.js
function arrayManipulation(n, queries) {
	const arr = new Array(n).fill(0)
    queries.forEach(([a,b,k]) => {
        arr[a-1] += k
        if(b < n) arr[b] -= k
    })
   console.log(arr.reduce(([c,mx],el) => [c += el, c > mx ? c : mx],[0,0])[1])
}

arrayManipulation(10, [[1,5 ,3],[4, 8, 5]] )